ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -55,25 +55,21 @@ - (void)acquireToken:(NSString *)apiId
     
     __block NSString* log = [NSString stringWithFormat:@"##### BEGIN acquireToken%@ (authority = %@, resource = %@, clientId = %@, idtype = %@) #####",
                              _silent ? @"Silent" : @"", _requestParams.authority, _requestParams.resource, _requestParams.clientId, [_requestParams.identifier typeAsString]];
-    AD_LOG_INFO_F(log, _requestParams.correlationId, @"userId = %@", _requestParams.identifier.userId);
+    AD_LOG_INFO_PII(_requestParams.correlationId, @"userId = %@", _requestParams.identifier.userId);
     
     ADAuthenticationCallback wrappedCallback = ^void(ADAuthenticationResult* result)
     {
-        NSString* finalLog = nil;
         if (result.status == AD_SUCCEEDED)
         {
-            finalLog = [NSString stringWithFormat:@"##### END %@ succeeded. #####", log];
+            AD_LOG_INFO(result.correlationId, @"##### END %@ succeeded. #####", log);
```

Comment:
log can have OII through resource and possibly authority, if authority is not a known host